src/main/java/org/broadinstitute/consent/http/db/InstitutionDAO.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.models.Institution;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.GetGeneratedKeys;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.sqlobject.transaction.Transactional;
+
+import java.util.Date;
+import java.util.List;
+
+public interface InstitutionDAO extends Transactional<InstitutionDAO> {
+
+  @SqlUpdate("insert into institution " +
+    "(institution_name, it_director_name, it_director_email, create_user, create_date) " +
+    "values (:institutionName, :itDirectorName, :itDirectorEmail, :createUser, :createDate)")
```

Comment:
We should standardize the sql. Capitalize keywords and keep the tables/columns lower case
```suggestion
  @SqlUpdate("INSERT INTO institution " +
    "(institution_name, it_director_name, it_director_email, create_user, create_date) " +
    "VALUES (:institutionName, :itDirectorName, :itDirectorEmail, :createUser, :createDate)")
```